### DISCUSSION TITLE
Just some pics

### DISCUSSION THREAD

**Discussion by terminator356** [muse-sequencer/muse#1198] [Team Posts]
And here is said Rideau Canal skateway:
![Canal_1](https://user-images.githubusercontent.com/5422362/105641681-6a152800-5e53-11eb-87ee-555e907a15e8.jpg)

Until last week there was no snow. People were getting really bored, 
 with the emergency lock-down in place.
Then... Big snow. People came out all over the place and made hundreds of snowmen.
Prompting some to observe there were more snowmen than humans in the parks!
Here is just one that shows how much time people have on their hands...
![giant_snowman_1](https://user-images.githubusercontent.com/5422362/105641692-7a2d0780-5e53-11eb-9a6d-2bfb0c314dce.jpg)

**Comment by spamatica** [comment]
Haha, nice, that is one huuge snowman! And the first pic is that ice for skating, very big too, a little bumpy right now it seems ;)

**Comment by terminator356** [comment]
Preparing the ice. Closed but opening soon.
At risk of a fine I wandered onto the ice a bit to get that shot.
Will send another when socially-distancing populated...
